terraform/groups/ecs-service/locals.tf

```diff
@@ -1,98 +1,70 @@
 # Define all hardcoded local variable and local variables looked up from data resources
 locals {
-  stack_name                = "filing-close" # this must match the stack name the service deploys into
-  name_prefix               = "${local.stack_name}-${var.environment}"
-  service_name              = "dissolution-web"
-  container_port            = "3000" # default node port required here until prod docker container is built allowing port change via env var
-  docker_repo               = "dissolution-web"
-  lb_listener_rule_priority = 21
-  lb_listener_paths         = ["/close-a-company","/close-a-company/","/close-a-company/*"]
-  healthcheck_path          = "/close-a-company/" #healthcheck path for dissolution web
-  healthcheck_matcher       = "200"
+  stack_name                  = "filing-close" # this must match the stack name the service deploys into
+  name_prefix                 = "${local.stack_name}-${var.environment}"
+  global_prefix               = "global-${var.environment}"
+  service_name                = "dissolution-web"
+  container_port              = "3000" # default node port required here until prod docker container is built allowing port change via env var
+  docker_repo                 = "dissolution-web"
+  kms_alias                   = "alias/${var.aws_profile}/environment-services-kms"
+  lb_listener_rule_priority   = 21
+  lb_listener_paths           = ["/close-a-company","/close-a-company/","/close-a-company/*"]
+  healthcheck_path            = "/close-a-company/" #healthcheck path for dissolution web
+  healthcheck_matcher         = "200"
+  vpc_name                    = data.aws_ssm_parameter.secret[format("/%s/%s", local.name_prefix, "vpc-name")].value
```

Comment:
I think the vpc_name is supposed to be read from Vault instead of AWS SSM Param store.
This entry is used only for the terraform infra purpose. Not for the application runtime purpose.

Few discussions here which can give some insight:
https://companieshouse.slack.com/archives/C05T4D1SS65/p1705404732491529?thread_ts=1705330292.595089&cid=C05T4D1SS65

Reply:
This is in the paramstore